src/test/java/study/StringTest.java

```diff
@@ -0,0 +1,96 @@
+package study;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.*;
+
+class StringTest {
+
+    @Test
+    @DisplayName("쉼표(,)로 구분된 숫자 문자열을 split하면 각 숫자가 배열로 반환된다.")
+    void stringSplitByComma() {
+
+        // given
+        String commaSeparatedNumbers = "1,2";
+
+        // when
+        String[] splitResult = commaSeparatedNumbers.split(",");
+
+        // then
+        assertThat(splitResult).contains("1", "2");
```

Comment:
```suggestion
        assertThat("1,2".split(",")).contains("1", "2");
```

로컬 변수 없이 구현해도 되지 않을까?
극단적으로 위와 같이 한 라인으로 구현해 보는 것은 어떨까?
극단적으로 짧게 구현도 해보면서 본인만의 스타일을 찾아보면 어떨까?